.github/workflows/build-and-push-containers.yml

```diff
@@ -11,65 +11,44 @@ jobs:
     env:
       DOCKER_USER: ${{ secrets.CONTAINER_USERNAME }}
     steps:
-      - name: Compile tag list
-        id: tags
-        run: |
-          TAG=${GITHUB_REF/refs\/tags\//}
-          PREFIX=ghcr.io/laminas/laminas-continuous-integration
-          MAJOR="${PREFIX}:$(echo ${TAG} | cut -d. -f1)"
-          MINOR="${MAJOR}.$(echo ${TAG} | cut -d. -f2)"
-          PATCH="${PREFIX}:${TAG}"
-          echo "::set-output name=tags::[\"${MAJOR}\",\"${MINOR}\",\"${PATCH}\"]"
-
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Setup QEMU
-        uses: docker/setup-qemu-action@v1
-
       - name: Setup Docker Buildx
         uses: docker/setup-buildx-action@v1
+        with:
+          install: true
 
       - name: Login to GitHub Container Registry
-        if: ${{ env.DOCKER_USER }}
+        if: ${{ github.event_name == 'release' }}
         uses: docker/login-action@v1
         with:
           registry: ghcr.io
-          username: ${{ env.DOCKER_USER }}
-          password: ${{ secrets.CONTAINER_PAT }}
-
-      - name: Build and push for release
-        if: ${{ github.event_name == 'release' }}
-        uses: docker/build-push-action@v2
-        with:
-          context: .
-          file: ./Dockerfile
-          platforms: linux/amd64
-          pull: true
-          push: true
-          tags: ${{ join(fromJSON(steps.tags.outputs.tags), ',') }}
-          cache-from: type=registry,ref=ghcr.io/laminas/laminas-continuous-integration-action:build-cache
-          cache-to: type=registry,ref=ghcr.io/laminas/laminas-continuous-integration-action:build-cache,mode=max
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build for local pull request
-        if: ${{ github.event_name == 'pull_request' && env.DOCKER_USER }}
-        uses: docker/build-push-action@v2
+      - name: Docker meta
+        id: docker_meta
+        uses: docker/metadata-action@v3
         with:
-          context: .
-          file: ./Dockerfile
-          platforms: linux/amd64
-          pull: true
-          push: false
-          cache-from: type=registry,ref=ghcr.io/laminas/laminas-continuous-integration-action:build-cache
-          cache-to: type=registry,ref=ghcr.io/laminas/laminas-continuous-integration-action:build-cache,mode=max
-
-      - name: Build for fork pull request
-        if: ${{ github.event_name == 'pull_request' && ! env.DOCKER_USER }}
+          images: ghcr.io/laminas/laminas-continuous-integration
+          tags: |
+            type=semver,pattern={{version}}
```

Comment:
`{{version}}` is shorthand for `{{major}}.{{minor}}.{{patch}}`
It uses tag ref associated with release to extract version info.

Reply:
semver is correct. `v` prefix is dropped and ignored